capitains_nautilus/cts/collections.py

```diff
@@ -168,8 +157,13 @@ class SparqlXmlCtsEditionMetadata(SparqlXmlCtsTextMetadata, XmlCtsEditionMetadat
 
 
 class SparqlXmlCtsWorkMetadata(CTSSparqlNavigatedCollection, XmlCtsWorkMetadata):
```

Comment:
`CTSSparqlNavigatedCollection` should probably be `CtsSparqlNavigatedCollection`